backend/src/apiserver/server/fakes_test.go

```diff
@@ -271,8 +271,9 @@ func initWithExperimentsAndTwoPipelineVersions(t *testing.T) *resource.FakeClien
 	resourceManager = resource.NewResourceManager(clientManager, &resource.ResourceManagerOptions{CollectMetrics: false})
 	_, err = resourceManager.CreatePipelineVersion(
 		&model.PipelineVersion{
-			Name:       "pipeline_version",
-			PipelineId: DefaultFakeUUID,
+			Name:         "pipeline_version",
+			PipelineId:   DefaultFakeUUID,
+			PipelineSpec: "apiVersion: argoproj.io/v1alpha1\nkind: Workflow",
```

Comment:
how is this related to this PR?

Reply:
If a mock PipelineSpec isn't provided, the Mock ResourceManager will attempt to fetch the spec from ObjectStore which will break as it no longer is uploaded there.  This behavior only exists in the test RM, I believe

Reply:
yep: https://github.com/kubeflow/pipelines/blob/fbb8b399b32e49f0683d2adf85d89dcbc30e3a31/backend/src/apiserver/storage/pipeline_store.go#L433 this is now fetched from the PipelineSpec column in the DB.